_guides/rust/command-line.md

```diff
@@ -0,0 +1,160 @@
+---
+title: Command Line Application
+date:  2020-05-03
+tags:
+  - command-line
+---
+
+Rust is a strongly typed programming language with focus on performance and memory-safety.
+It's been designed at Mozilla Labs in 2010.
+
+
+## Further Material
+
+- Homepage: [rust-lang.org](https://www.rust-lang.org)
+- The Rust Book: [doc.rust-lang.org](https://doc.rust-lang.org/book)
+- Rust Package Registry: [crates.io](https://crates.io)
+- Cargo Book: [doc.rust-lang.org](https://doc.rust-lang.org/cargo/guide)
+
+
+## Topics, Tools and Terms
+
+Rust packages are called _crates_, and the tool to manage the lifecycle of a project is called _cargo_.
+It comes with the Rust language distribution itself.
+
+Cargo lets us run commands to build and test projects or install new binaries.
+It does not provide a way via the command line to add a new dependency, though.
```

Comment:
That is true, but the community built https://github.com/killercup/cargo-edit which does allow you to do `cargo add foo` and `cargo remove bar`

Reply:
Good point, I think I should move line 33 up here then. I think up here is where that line meant to be anyways, as it's pretty out of context down there talking about cargo-edit. :)